src/session.rs

```diff
@@ -1072,6 +1073,34 @@ impl Session {
         self.view_coords(self.views.active_id, p)
     }
 
+    /// Expand paths, e.g. by converting ~ into the user's home directory path
+    pub fn expand_path<P: AsRef<Path>>(&mut self, path: P) -> OsString {
```

Comment:
I don't think `self` needs to be mutable here.

Reply:
Fixed.